frontend/src/metabase/dashboard/hoc/withAutoApplyFilters.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useCallback, useEffect, useState } from "react";
+import _ from "underscore";
+
+import { Dashboard } from "metabase-types/api";
+import { ApplyButton } from "./withAutoApplyFilters.styled";
+
+// XXX: Use similar type as in https://github.com/metabase/metabase/blob/c56d0872ac2a2cf44a827acfa1af5e8b76dcaa7d/frontend/src/metabase-lib/parameters/utils/cards.ts#L17
+type ParameterValues = Record<string, any>;
```

Comment:
Use similar type as in https://github.com/metabase/metabase/blob/c56d0872ac2a2cf44a827acfa1af5e8b76dcaa7d/frontend/src/metabase-lib/parameters/utils/cards.ts#L17